komodo-core-tests/src/test/resources/org/komodo/repository/test/test-local-repository-on-file-config.json

```diff
@@ -84,6 +77,9 @@
         "removeDerivedContentWithOriginal" : true,
         "threadPool" : "modeshape-workers",
         "sequencers" : {
+            "VDB Dynamic Sequencer" : {
+				"classname" : "org.teiid.modeshape.sequencer.vdb.VdbDynamicSequencer"
```

Comment:
Is this necessary? Only ask since the KDdlSequencer already depends upon it and the other sequencers, like the DDL or TSQL, haven't been included here.